babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -841,6 +846,8 @@ SqlNode DateTimeTerm() :
         e = DateTimeLiteral()
     |
         e = SimpleIdentifier()
+    |
+        e = DateFunctionCall()
```

Comment:
DateFunctionCall is also in the [BuiltinFunctionCalls](https://github.com/googleinterns/calcite/blob/master/babel/src/main/codegen/config.fmpp#L963). Does it need to be called in both of these places?

Reply:
It does. This one is to allow for DATE AT LOCAL queries and the one in BuiltinFunctionCalls is for standalone DATE queries.

Reply:
Ah ok, how come the first call to DateFunctionCall() wouldn't parse both of these?

Reply:
I figured that it would be simplest to just use the existing parsing logic for AT LOCAL rather than add more logic (for a fairly edge case too) to DateFunctionCall().

Reply:
Oh I see, that makes sense